### PR TITLE
Adding support for the container scan option 

### PR DESCRIPTION
**Pull request by dstiliadis**
Adds the --container scan option to the action parameters. Documentation will follow in subsequent PRs.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -21941,6 +21947,13 @@ function get_scan_options(options) {
     if (SCAN_SECRETS) {
         options.push(`--secrets=true`);
     }
+    if (SCAN_CONTAINER) {
+        if (!SCAN_PROJECT_NAME) {
```
Comment (IAmATeaPot418) [inline dist/index.js]: image name must be provided as well?

File: src/scan.ts [hunk 2]
```diff
@@ -45,6 +53,13 @@ function get_scan_options(options: any[]): void {
   if (SCAN_SECRETS) {
     options.push(`--secrets=true`);
   }
+  if (SCAN_CONTAINER) {
+    if (!SCAN_PROJECT_NAME) {
+      core.error("Project name must be provided with scan_container");
```
Comment (IAmATeaPot418) [inline src/scan.ts]: image name?

**Comment by peter-cg** [comment]
@dstiliadis you want to merge this or wait for repo association support?

File: dist/index.js (hunk 1 above)
Comment (upadhyayap) [reply dist/index.js]: @IAmATeaPot418  Added that check.

File: src/scan.ts (hunk 2 above)
Comment (upadhyayap) [reply src/scan.ts]: Added check for image.

**Comment by upadhyayap** [comment]
> @dstiliadis you want to merge this or wait for repo association support?

I have updated the PR according to the latest changes in endorctl. Now `--project-name` is not mandatory. 
@peter-cg   @IAmATeaPot418 Can you please take a look again.

File: action.yml [hunk 3]
```diff
@@ -108,6 +108,18 @@ inputs:
     description: >-
       "Scan git commits and generate findings for all dependencies."
     default: true
+  # Scan container images
+  scan_container:
+    description: >-
+      "Scan a specified container image. The image must be set with `image` and a project can be defined with `project`."
```
Comment (hghmn) [inline action.yml]:
```suggestion
      "Scan a specified container image. The image must be set with `image` and a project can be defined with `project_name`."
```

**Review by hghmn — DISMISSED** [review]
Tested successfully with `project_name` and with the default `path`

**Review by hghmn — APPROVED** [review]

**Pull request merged**